.github/CONTRIBUTING.md

```diff
@@ -8,7 +8,6 @@ pay attention to a few things:
 3. your code must be documented
 4. your work must be signed (see below)
 5. you may contribute through GitHub Pull Requests on `master` branch
-6. the sentences must be done in French (temporarily, due to internal process)
```

Comment:
Why did you remove this ?

Reply:
This generator doesn't rely on any translations files.

I suspect a copy/paste from previous `ovh-ux/ovh-manager-*` repositories.